test/fixtures/rules/vue/vue.js

```diff
@@ -0,0 +1,10 @@
+// `vue/html-quotes` - enforce double quotes
```

Comment:
Vue's ESLint plugin [has many, many more rules](https://eslint.vuejs.org/rules/). Shouldn't we add fixtures for other key rules as well? 🤔

Reply:
It does 😅 
Guess I'll just stop being lazy and do them all

Reply:
Done 😓